src/content/docs/apm/agents/manage-apm-agents/app-naming/use-multiple-names-app.mdx

```diff
@@ -15,36 +15,40 @@ redirects:
   - /docs/apm/new-relic-apm/installation-configuration/use-multiple-names-app
 ---
 
-import apmInsightsToggleonandOff from 'images/apm_screenshot-full_insights-toggle-on-and-off.png'
```

Comment:
@zuluecho9 Is there nothing we can replace this image with?

Reply:
no, it's not really comparable anymore